### ISSUE TITLE
Momentum methods

### ISSUE THREAD

**Issue by isaachenrion** [isaachenrion/dl-nyu#5]

**Comment by isaachenrion** [comment]
Results after 10 epochs of training (with batch normalization)
- SGD, lr=0.01, momentum=0.5: 90%
- SGD, lr=0.01, momentum=0.0: 89%
- Adam, lr=0.001: 94%
- RMSprop, lr=0.01: 97%
